simulator/mips/mips_instr.h

```diff
@@ -350,9 +373,10 @@ class MIPSInstr
         {
             return (operation == OUT_R_ACCUM) ? 1 : (operation == OUT_R_SUBTR) ? -1 : 0;
         }
+
         bool is_store() const { return operation == OUT_I_STORE  ||
-                                       operation == OUT_I_STORER ||
-                                       operation == OUT_I_STOREL; }
+                                       operation == OUT_I_PARTIAL_STORE; }
```

Comment:
I suggest to use "is_partial_store()" here. When someone adds a new operation code (e.g. for 64 bit stores), he/she needs to change only one line of code.